name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/wirbank/WirBankPDFExtractorTest.java

```diff
@@ -807,6 +807,56 @@ public void testWertpapierKauf10WithSecurityInCHF()
                         }))));
     }
 
+    @Test
+    public void testWertpapierKauf11()
+    {
+        Client client = new Client();
+
+        WirBankPDFExtractor extractor = new WirBankPDFExtractor(client);
+
+        List<Exception> errors = new ArrayList<>();
+
+        List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "Kauf11_Francais.txt"), errors);
+
+        assertThat(errors, empty());
+        assertThat(results.size(), is(2));
+        new AssertImportActions().check(results, "CHF");
+
+        // check security
+        Security security = results.stream().filter(SecurityItem.class::isInstance).findFirst()
+                        .orElseThrow(IllegalArgumentException::new).getSecurity();
+        assertThat(security.getIsin(), is("CH0030849613"));
+        assertNull(security.getWkn());
+        assertNull(security.getTickerSymbol());
+        assertThat(security.getName(), is("CSIF Canada"));
+        assertThat(security.getCurrencyCode(), is(CurrencyUnit.CAD));
```

Comment:
```suggestion
        assertThat(security.getCurrencyCode(), is("CAD"));
```

We have only defined two CurrencyUnit as fixed constants. Once EUR and once USD. All other currencies are to be set as strings.